Services/Membership/classes/Export/class.ilMemberExportGUI.php

```diff
@@ -87,7 +87,7 @@ public function executeCommand() : void
     protected function initSettingsForm(bool $a_is_excel = false) : ilPropertyFormGUI
     {
         // Check user selection
-        $this->exportSettings = new ilUserFormSettings('memexp');
+        $this->exportSettings = new ilUserFormSettings('memexp'); // TODO PHP8-REVIEW Expected parameter of type 'int', 'string' provided
```

Comment:
// TODO PHP8-REVIEW Expected parameter of type 'int', 'string' provided

Reply:
This and all other calls of ilUserFormSettings were a wrong refactoring in Services/User. I fixed this in the trunk.
Now the expected first parameter is "string"